io/src/lzf.cpp

```diff
@@ -90,7 +90,7 @@ unsigned int
 pcl::lzfCompress (const void *const in_data, unsigned int in_len,
                   void *out_data, unsigned int out_len)
 {
-  LZF_STATE htab;
+  LZF_STATE htab{};
```

Comment:
Are you sure this will 0 init (for original L106)

Reply:
Per [CPPReference](https://en.cppreference.com/w/cpp/language/zero_initialization):

> If T is array type, each element is zero-initialized.